### ISSUE TITLE
enableIPForwarding  lost on failover

### ISSUE THREAD

**Issue by chen23** [F5Networks/f5-azure-arm-templates#43]
## Description
When running failoverProvider.js the external interface will lose the setting for whether IPForwarding is enabled on the interface.
## Template
ha-avset existing

"f5-cloud-libs-azure",  "version": "1.3.0",

## Severity Level

Severity: 3

The following is an example of how to remedy.

```
--- failoverProvider.js.orig    2017-11-29 00:01:45.340480857 -0500
***************
*** 344,354 ****
      var myNicArr = [];

      var ourLocation;
      var theirNsg;
      var myNsg;
-     var enableIPForwarding;

      var associateArr = [];
      var disassociateArr = [];

      var retryDissassociateNics = function(resourceGroup, theirNicName, theirNicParams) {
--- 344,353 ----
***************
*** 466,478 ****
      }

      ourLocation = myNicConfig.location;
      theirNsg = theirNicConfig.networkSecurityGroup;
      myNsg = myNicConfig.networkSecurityGroup;
!     enableIPForwarding = myNicConfig.enableIPForwarding
      theirNicParams = { location: ourLocation, ipConfigurations:theirNicArr, networkSecurityGroup: theirNsg };
!     myNicParams = { location: ourLocation, ipConfigurations:myNicArr, networkSecurityGroup: myNsg, enableIPForwarding: enableIPForwarding };

      disassociateArr = [resourceGroup, theirNicName, theirNicParams];
      associateArr = [resourceGroup, myNicName, myNicParams];

      util.tryUntil(this, {maxRetries: 4, retryIntervalMs: 15000}, retryDissassociateNics, disassociateArr)
--- 465,477 ----
      }

      ourLocation = myNicConfig.location;
      theirNsg = theirNicConfig.networkSecurityGroup;
      myNsg = myNicConfig.networkSecurityGroup;
!
      theirNicParams = { location: ourLocation, ipConfigurations:theirNicArr, networkSecurityGroup: theirNsg };
!     myNicParams = { location: ourLocation, ipConfigurations:myNicArr, networkSecurityGroup: myNsg };

      disassociateArr = [resourceGroup, theirNicName, theirNicParams];
      associateArr = [resourceGroup, myNicName, myNicParams];

      util.tryUntil(this, {maxRetries: 4, retryIntervalMs: 15000}, retryDissassociateNics, disassociateArr)
```

**Comment by edwoodjrjr** [comment]
Will fold this into an upcoming release.

**Comment by chen23** [comment]
Thanks. Disassociate also needs to be updated as well.

**Comment by chen23** [comment]
Also may be worth enabling ipforwarding by default to match the internal interface.

**Comment by jsevedge** [comment]
This is now resolved in v5.0.0.0, ha-avset is now located under cluster/failover-api.
